app/src/main/java/org/wikipedia/readinglist/ReadingListActivity.java

```diff
@@ -20,6 +20,8 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         getWindow().clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
         getWindow().setStatusBarColor(Color.TRANSPARENT);
+        // TODO: translucent status will affect the navigation bar color?
+        getWindow().setNavigationBarColor(0);
```

Comment:
What does `0` mean here? And why are we calling `getWindow().setNavigationBarColor()` directly, and not our own `BaseActivity.setNavigationBarColor()`?

Reply:
Fixed and done.